web/impact/impact/models/utils.py

```diff
@@ -2,8 +2,28 @@
 # Copyright (c) 2017 MassChallenge, Inc.
 
 
+import re
+
+
 LABEL_LENGTH = 255
 
 
 def is_managed(db_table):
     return False
+
+
+def camel_case_to_snake(value):
+    original_model_string = re.findall('[A-Z][^A-Z]*', value)
```

Comment:
This regex fails to catch a string that starts with lowercase characters. Not sure if we care, but it's worth noting that official camelCase is considered to start with lower-case characters for the first words (https://en.wikipedia.org/wiki/Camel_case).

Reply:
And you got to love the irony in that wikipedia entry.

Reply:
Name changed.